### DISCUSSION TITLE
Please create a plugin that can create permisions per field

### DISCUSSION THREAD

**Discussion by Rich3art** [Budibase/budibase#8285] [Ideas]
Hi,

Please create a plugin that I will be able to have a permision for that specific field and not only for the whole app/database.

In other words: lets say there are employees that should only be able to enter certain info on the form and then managers should be able to enter ALL info on the form

**Comment by kasgithub8** [comment]
Just disable those fields with a role condition

**Reply by melohagan** [reply]
This may help as well! https://docs.budibase.com/docs/conditions#showing-and-hiding-a-component-based-on-the-users-role
